src/Engine/AssetsManager.h

```diff
@@ -33,6 +33,8 @@ class AssetsManager {
     Texture *GetSprite(const std::string &name, unsigned int palette_id,
                        unsigned int lod_sprite_id);
 
+    Texture* WinnerCert{ nullptr };
```

Comment:
TBH I don't think this field really belongs here, let's add a TODO to move it.

Reply:
All "loose" textures like this (and minimap and font atlas ect) should be tracked in asset manager. Will leave a todo for better containment.